troubadix/helper/patterns.py

```diff
@@ -87,12 +87,29 @@ class ScriptTag(Enum):
 __script_tag_pattern = None
 
 __DATE_VALUE = r"[A-Za-z0-9\:\-\+\,\s\(\)]{44}"
+__CVSS_V2_BASE_VECTOR = (
+    r"AV:[LAN]/AC:[HML]/Au:[NSM]/C:[NPC]/I:" r"[NPC]/A:[NPC]"
```

Comment:
The strings are short enough to be combined into a single one and fit into the line
```suggestion
    r"AV:[LAN]/AC:[HML]/Au:[NSM]/C:[NPC]/I:[NPC]/A:[NPC]"
```